### PR TITLE
feat(7): add video description generation

### LINKED ISSUE #7: Feature: Add video description generation
## Is your feature request related to a problem? Please describe.

Users processing the videos are finding that creating the descriptions for the videos are tedious

## Describe the solution you'd like

Add a feature to generate the full description with minimal effort

## Additional context

We can utilize studylight's api to fetch the NIV 1984 text

### PR DESCRIPTION
**Pull request by paulhchoi**
Closes #7

## Description

This PR:
- adds the ability to generate the YouTube title and description based on the values configured in the `.env` file.
- refactors the `config.txt` configration file to use `.env` files and the `python-dotenv` package to access `env` values.
- adds a startup script at `bin/start.sh`, so users don't have to run the docker commands directly.

## How to Test

Users can test this by:
- copying and adding to the new `.env` file (secrets in org bitwarden vault)
- using "smaller video" test values such as:
```
URL="https://www.youtube.com/watch?v=mjjp0dnX3Xc"
START="00:00:30"
END="00:01:00"

SERMON_TITLE="Faith"
PREACHER="Reverend Donny Cho"
PASSAGE="Isaiah 40:27-31"
SERIES="Our Functional Doctrines"
```

- running the new startup script with `bin/start.sh`
- selecting `d` for `[d]escription` when prompted
- looking into the `data/` directory for the resulting `generated_title.txt` and `generated_description.txt` files

### CODE REVIEW

**Comment by paulhchoi** [comment]
@13jkim mind giving this a run when you get a chance? should be good but the description generation is pretty jank bc the data the studylight api returns in super inconsistent. took me longer than im proud of to write the formatter for it lol.

**Comment by paulhchoi** [comment]
@dhy6785 @ellisPae tagged y'all bc this actually switches up a decent amount, from the `config` file to `env`, and the new startup script — will be good to step through the readme line by line next audio scrub session

File: utils/media.py [hunk 1]
```diff
@@ -82,7 +82,6 @@ def download_media_from_youtube(
     youtube_url: str,
     start_time: str,
     end_time: str,
-    upload_date: str,
```
Comment (paulhchoi) [inline utils/media.py:85]: 🤦‍♂️

File: utils/media.py [hunk 2]
```diff
@@ -91,7 +90,7 @@ def download_media_from_youtube(
     # Define filename based on media type
     media_filename = os.path.join(
         "tmp",
-        f"{upload_date}_base_downloaded_raw.{'mp3' if media_type == 'audio' else 'mp4'}",
```
Comment (paulhchoi) [inline utils/media.py:94]: @13jkim yup I was just looking at wrong code, I guess I did remove it here lol

**Review by 13mikj — APPROVED** [review]
lgtm w/ updated func signature

**Pull request merged**